tests/ui/const-generics/const_trait_fn-issue-88433.rs

```diff
@@ -16,7 +16,7 @@ struct Closure;
 impl const Func<&usize> for Closure {
     type Output = usize;
 
-    fn call_once(self, arg: &usize) -> Self::Output {
+    const fn call_once(self, arg: &usize) -> Self::Output {
```

Comment:
> all I did in this PR was to require `(const)` on trait methods of `#[const_trait]` traits and `const` on methods in `impl const Trait for Type`

Given that the tentative plan was to require `(const)` on the impl methods in some cases, such as the one below, would it be better to (at this point) require that in all cases?

```rust
#![feature(const_trait_impl)]

#[const_trait]
trait Tr {
  (const) fn f();
}

struct W<T>(T);
impl<T: ~const Tr> const Tr for W<T> {
  const fn f() { // <-- `(const)` required
    _ = T::f();
  }
}

fn main() {}
```

Reply:
Uhm. I was not aware of this plan and have never seen this suggestion. Please start a thread on the rfc

Reply:
OK:

- https://github.com/rust-lang/rfcs/pull/3762/files#r2046363055

The text of the RFC does already demonstrate this syntax.